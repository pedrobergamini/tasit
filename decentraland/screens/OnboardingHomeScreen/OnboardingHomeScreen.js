import React from "react";
import OnboardingHome from "@presentational/OnboardingHome";

export default class OnboardingHomeScreen extends React.Component {
  render() {
    return (
      <OnboardingHome
        onPress={() => this.props.navigation.navigate("EthereumQuestionScreen")}
      />
    );
  }
}
