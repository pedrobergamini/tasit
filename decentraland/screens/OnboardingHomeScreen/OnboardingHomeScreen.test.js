import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import OnboardingHomeScreen from "./OnboardingHomeScreen";

describe("OnboardingHome", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    expect(shallow(<OnboardingHomeScreen />)).toMatchSnapshot();
  });
});
