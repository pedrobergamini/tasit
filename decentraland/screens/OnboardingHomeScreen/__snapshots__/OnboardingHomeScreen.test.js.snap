// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnboardingHome renders the component 1`] = `
<OnboardingHome
  onPress={[Function]}
/>
`;
